ostd/src/sync/rcu/mod.rs

```diff
@@ -381,20 +363,12 @@ impl<P: OwnerPtr> RcuReadGuard<'_, P> {
 }
 
 // RCU guards that may have a null pointer can be dereferenced after checking.
```

Comment:
```
// RCU guards that have a non-null pointer can be directly dereferenced.
[..]
// RCU guards that may have a null pointer can be dereferenced after checking.
```

It is unfortunate that we have to remove `Deref`, but I have no better idea. So we remove the first comment. Maybe remove the second comment as well, since the two comments seem to work together?